spec/workers/rss_reader_fetch_user_worker_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe RssReaderFetchUserWorker, type: :worker do
+  let(:worker) { subject }
+
+  include_examples "#enqueues_on_correct_queue", "medium_priority", [456]
```

Comment:
Is `456` a hardcoded number or does it come from somewhere? Does it map to a user_id in our test infrastructure somewhere?

Reply:
Here it could be any integer here. The arguments passed to the worker are not actually used here because the worker itself won't be executed, this test just checks that it it added to the queue. You can check out the shared example code in `spec/workers/shared_examples/enqueues_on_correct_queue.rb`.
Maybe it's worth to replace a number with a random one for clarity.

Reply:
Updated it to `9999`!

Reply:
I feel like we could either just use the same number as the id (`1`?) or use `rand(1..999)` for example, which would really allow us to test _any_ integer and make it clear that it doesn't matter what it is, and it should work regardless of what we pass in. My worry with hardcoded integers is that someone might find them later on and not know what they mean (like me, lol).

Reply:
Sure thing - updated.

To my understanding, the original `456` was the random number that is used in the other worker specs.

My concern is that all the other worker specs that have this same test don't do that (`1` or `rand(1..999)`, so I'm worried the inconsistency might be confusing in the future.

I added `rand(1..999)` with a comment to explain it. Best of both worlds?

Reply:
After chatting with @atsmith813, we both agreed that it makes sense to stick with the hardcoded integer as the solution (with maybe a comment added), though we both want to think about how to move away from this in the future 👍